server.js

```diff
@@ -88,8 +89,9 @@ server.route([
           id: Joi.string(),
           frequency: Joi.string().valid("monthly", "one-time"),
           currency: Joi.string().min(3).max(3),
-          quantity: Joi.string(),
-          amount: Joi.number()
+          quantity: Joi.number(),
```

Comment:
quantity is actually passes as a number, and not a string. Fixing. This is busted on staging.